data/json/items/armor/jewelry.json

```diff
@@ -1471,8 +1459,7 @@
     "bashing": 4,
     "material": [ "fur", "plastic" ],
     "symbol": "s",
-    "color": "brown",
-    "flags": [ "FANCY" ]
+    "color": "brown"
```

Comment:
I'd put back FANCY but also add a bit of encumbrance (it getting tangled up with things!).